<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chartMenu</class>
 <widget class="QMainWindow" name="chartMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>chartMenu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Chart Examples</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>171</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="barChartBtn">
       <property name="text">
        <string>Bar Chart</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="lineChartBtn">
       <property name="text">
        <string>Line Chart</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pieChartBtn">
       <property name="text">
        <string>Pie Chart</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>293</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
